src/js/config.js

```diff
@@ -18,9 +18,9 @@ module.exports = {
     text_for_map_search: "",
   },
 
-  FACEBOOK_APP_ID: "",
+  FACEBOOK_APP_ID: "1097389196952441",
 
```

Comment:
Hi @SailingSteve, could you please remove this config.js file from your pull request?